index.html

```diff
@@ -0,0 +1,111 @@
+  </head>
+  <body>
+    <div class="header-container">
+      <div class="toolbar">
+        <div class="logo">
+          <p>Rose Okpe</p>
+        </div>
+        <nav>
+          <div>
+            <a class="nav-button" href="#"
+              ><img src="images/home-icon.png" alt=""
+            /></a>
+          </div>
+        </nav>
+      </div>
+    </div>
+
+    <!--    section     -->
+    <div class="section-container">
+      <div class="headline">
+        <div class="primary-text">
+          <h1>Rose Okpe</h1>
+        </div>
+        <div class="support-text">
+          <p>Software Developer</p>
+          <p></p>
+        </div>
+
+        <div class="intro-text">
+          <p>
+            I can help you build a product,<br />feature or website look through
+            <br />some of my work and experience! if<br />you like what you see
+            and have a <br />
+            project you need coded, don't <br />
+            hestitate to contact me.
+          </p>
+        </div>
+        <hr />
+        <div class="frame38-container">
+          <div class="languages">
+            <h4>Languages</h4>
+            <div class="language-list">
+              <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </div>
+          </div>
+          <div class="frameworks">
+            <h4>Frameworks</h4>
+            <div class="framework-list">
+              <ul>
+                <li>Bootsrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+               </ul>
+            </div>
+          </div>
+          <div class="skills">
+            <h4>Skills</h4>
+            <div class="list-skills">
+              <ul>
+                <li>Codekit</li>
+                <li>Githubt</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+               </ul>
+              </div>
+          </div>
+        </div>
+        <hr />
+        <div class="support-item">
+          <div class="social-media">
+            <a href=""
+              ><img class="git-icon" src="images/git-icon.png" alt=" "
+            /></a>
+            <a href=" "
+              ><img class="linkedin-icon" src="images/Linkedin-icon.png" alt=""
+            /></a>
+            <a href=""
+              ><img class="angelist-icon" src="images/Icon-Angelist.png" alt=""
+            /></a>
+            <a href=""
+              ><img class="Twitter-icon" src="images/Icon-Twitter.png" alt=""
+            /></a>
+            <a href=""
+              ><img class="Medium-icon" src="images/Medium-icon.png" alt=""
+            /></a>
+          </div>
+          <div class="element-right">
+            <a href=""><img src="images/Group-55.png" alt="" /></a>
+            <p>Work</p>
```

Comment:
[ OPTIONAL ] The list of icons should use a `ul` tag to group them, and `li` tag per icon; each icon should have a link `a` tag.